lib/src/rdb/datastore.rs

```diff
@@ -418,6 +418,39 @@ impl RocksdbDatastore {
 
         let metadata_manager = MetadataManager::new(&db);
         let indexed_properties = metadata_manager.get_indexed_properties()?;
+        drop(metadata_manager);
```

Comment:
There's no point in dropping this early AFAICS

Reply:
This line is for MultiThreaded DB (`multi-threaded-cf`
When without this line:

```bash
error[E0505]: cannot move out of `db` because it is borrowed
   --> lib/src/rdb/datastore.rs:423:26
    |
419 |         let metadata_manager = MetadataManager::new(&db);
    |                                                     --- borrow of `db` occurs here
...
423 |             db: Arc::new(db),
    |                          ^^ move out of `db` occurs here
...
426 |     }
    |     - borrow might be used here, when `metadata_manager` is dropped and runs the destructor for type `managers::MetadataManager<'_>`
```

Reply:
Im agree to delete

Reply:
I think this can be fixed now?